keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1061,16 +1062,37 @@ Details to consider:
 
 #### CEL Expression Composition
 
+##### Use Cases
+
+###### Reusing/memoizing an expensive computation
+
+For a CEL expression that runs in O(n^2) time or worse (or otherwise
+takes a significant amount of time to execute), it would be nice to only run
+it when necessary. For instance, if multiple validation expressions used the
```

Comment:
Lead with the multiple validation expression case since that is the case that matter? (Strictly speaking, it would be possible to do this optimization today since CEL is purely functional. I.e. the results of identical sub-expressions could be memorized today in single CEL expressions).

Reply:
Right, I've re-ordered the use cases section.